packages/app/src/components/VisualizationOptions/Options/Outliers.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import i18n from '@dhis2/d2-i18n'
+import { Checkbox, FieldSet, Help, Legend } from '@dhis2/ui'
+
+import { sGetUiOptions } from '../../../reducers/ui'
+import { acSetUiOptions } from '../../../actions/ui'
+import {
+    tabSectionOptionToggleable,
+    tabSectionOption,
+    tabSectionTitle,
+} from '../styles/VisualizationOptions.style.js'
+import OutlierDetectionMethod from './OutlierDetectionMethod'
+import ExtremeLines from './ExtremeLines'
+import styles from '../styles/Outliers.module.css'
+
+const ENABLED_PROP = 'enabled'
+const METHOD_PROP = 'method'
+const THRESHOLD_PROP = 'thresholdFactor'
+const EL_ENABLED_PROP = 'enabled'
+const EL_VALUE_PROP = 'value'
+
+const OUTLIER_ANALYSIS_OPTION_NAME = 'outlierAnalysis'
+
+const METHOD_IQR = 'IQR'
+const METHOD_STANDARD_Z_SCORE = 'STANDARD_Z_SCORE'
+const METHOD_MODIFIED_Z_SCORE = 'MODIFIED_Z_SCORE'
+const methods = [
+    {
+        id: METHOD_IQR,
+        label: i18n.t('Interquartile Range (IQR)'),
+        defaultThreshold: 1.5,
+    },
+    {
+        id: METHOD_STANDARD_Z_SCORE,
+        label: i18n.t('Z-score / Standard score'),
+        defaultThreshold: 3,
+    },
+    {
+        id: METHOD_MODIFIED_Z_SCORE,
+        label: i18n.t('Modified Z-score'),
+        defaultThreshold: 3,
+    },
+]
+
+const DEFAULT_STATE = {
+    [ENABLED_PROP]: false,
+    [METHOD_PROP]: METHOD_IQR,
+    [THRESHOLD_PROP]: 1.5,
```

Comment:
What about `normalizationMethod`?

Reply:
According to @janhenrikoverland we will hardcode the `normalizationMethod` to `XY_RATIO` for now and the prop on the backend is only for future use.

Reply:
👍